accessibility/idl$ rm -r ../unxlngi4.pro/ ; (cd ../idl/org/openoffice/accessibility/internal/; dmake) ; dmake -P4
dmake:  Error -- `../unxlngi4.pro/misc/java/org/openoffice/accessibility/internal/AccessibleContextInfo.java' not found, and can't be made

--- ooo_1.1beta2_src/accessibility/idl/makefile.mk.orig	2003-05-27 21:54:24.000000000 +0200
+++ ooo_1.1beta2_src/accessibility/idl/makefile.mk	2003-05-27 22:02:36.000000000 +0200
@@ -109,6 +109,7 @@
 
 # Disable for building dependencies
 .IF "$(JAVACLASSFILES)" != ""
+$(JAVAFILES) : $(GENFLAGFILE)
 $(JAVACLASSFILES) : $(GENFLAGFILE)
 .ENDIF
 	
